helix-core/src/transaction.rs

```diff
@@ -754,4 +755,21 @@ mod test {
         use Operation::*;
         assert_eq!(changes.changes, &[Insert("a".into())]);
     }
+
+    #[test]
+    fn combine_with_utf8() {
+        const TEST_CASE: &'static str = "Hello, これはヒレクスエディターです！";
```

Comment:
Unrelated to code, but I expected ヘリックス here :laughing: 
```suggestion
        const TEST_CASE: &'static str = "Hello, これはヘリックスエディターです！";
```

Reply:
My 日本語 is not 上手 ;-;